molecule_ec2/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/converge.yml

```diff
@@ -2,6 +2,14 @@
 - name: Converge
   hosts: all
   tasks:
+    - name: Wait for boot process to finish
```

Comment:
we already have a place for these kind of thing. which is prepare.yml

Reply:
Thanks for the review. Should I move this block to `prepare.yml`?

Reply:
yes, i think so 
```
- name: Wait for cloud init to finish
      cloud_init_data_facts:
        filter: status
      register: res
      until: "res.cloud_init_data_facts.status.v1.stage is defined and not res.cloud_init_data_facts.status.v1.stage"
      retries: 100
      delay: 10
```